rialto_airflow/mais.py

```diff
@@ -34,22 +25,11 @@ class TokenFetchError(LookupError):
 def get_token(client_id: str, client_secret: str, base_url: str) -> str:
     """Retrieves an OAuth2 access token."""
 
-    token_url = f"{base_url}/api/oauth/token"
-    auth = f"{client_id}:{client_secret}"
-
-    headers = {
-        "Content-Type": "application/x-www-form-urlencoded",
-        "Authorization": f"Basic {auth}",
-    }
-
-    data = {
-        "grant_type": "client_credentials",
-        "client_id": client_id,
-        "client_secret": client_secret,
-    }
+    token_url = f"{base_url}/oauth2/token"
+    data = {"grant_type": "client_credentials"}
 
     try:
-        response = requests.post(token_url, headers=headers, data=data)
+        response = requests.post(token_url, auth=(client_id, client_secret), data=data)
         response.raise_for_status()  # Raise HTTPError for bad responses (4xx or 5xx)
```

Comment:
this was not working with their new API scheme, and this fixes it ... also, we didn't need to be passing in the client_id and client_secret in two places (header and payload) anyway